testing/regress/ecl/parquetDecimals.ecl

```diff
@@ -37,9 +37,9 @@ decimal_data := DATASET([{(DECIMAL) '0.12345678901234567890123456789012',
                           }], layout);
 
 overwriteOption := TRUE;
-ParquetIO.Write(decimal_data, '/home/jack/dev/debug/etc/HPCCSystems/mydropzone/regress/parquet/decimal.parquet', overwriteOption);
+ParquetIO.Write(decimal_data, '/etc/HPCCSystems/mydropzone/regress/decimal.parquet', overwriteOption);
```

Comment:
Better, but I don't think this is the correct path though.  I think it should be

/var/lib/HPCCSystems/mydropzone/regress/...

Also, that assumes that the system has been installed in the root directory - that isn't the case for many developers.  @jakesmith can you think of a clean way of doing this?

Reply:
@jackdelv - you could use Std.File.GetDefaultDropZone (or potentially Std.File.GetLandingZones if you needed >1 and/or more detail)

Reply:
Changed to use Std.File.GetDefaultDropZone to prefix the file name with a valid dropzone directory.